pom.xml

```diff
@@ -148,7 +147,6 @@
 	<modules>
     	<module>mars-sim-tools</module>
 		<module>mars-sim-mapdata</module>
-		<module>mars-sim-main</module>
```

Comment:
So, the maven submodule mars-sim-main have no use now ?

Reply:
Correct